specs/pluggable_alert_delivery.adoc

```diff
@@ -195,13 +259,15 @@ Primary assignee:
 == Documentation impact
 
 * This adds 2 apis:
-  ** One to list supported alert notifying mechanisms.
-  ** One for Users to subscribe to interested alerts and be notified by chosen
-     alerting means.
+  ** Api to get list of currently supported means of notification.
+  ** Api to get list of alerts with various filtering options such as based on
+     time, acked/not acked, alert type, severity, resource and significance.
+
 
 == Testing
 
-This adds new apis for user configuration which need to be tested.
+This spec introduces an api to list available means of alert notification which
+needs to be tested.
 
 == References
```

Comment:
Issues linked to this spec are not correct. Need to:
- raise an issue on specification repository
- raise issues on all the affected components and link it here